src/main/resources/pdf-map.yaml

```diff
@@ -12,35 +12,38 @@ inputFields:
   # Green lines = in YAML but not PDF
 
   # Vraj - Usable from here
-  applicantFullName: applicantFullName
-  phoneNumber: homePhone
-  speaksEnglish: speaksEnglish
-  applicantSSN: applicantSSN
-  applicantDOB: applicantDOB
-  homeAddressStreetAddress1: homeAddressNumberAndStreet
-  homeAddressStreetAddress2: homeAddressStreetAddress2
-  homeAddressZipCode: homeAddressZipCode
-  homeAddressState: homeAddressState
-  homeAddressCity: homeAddressCity
-  mailingAddress: mailingAddress
-  signature: signature
-  sex: applicantSex
-  isApplicantApplying: isApplicantApplying
-  citizenshipStatus: isApplicantCitizen
-  someoneInHHIsPregnant: someoneInHHIsPregnant
```

Comment:
someoneInHHIsPregnant, mailingAddress, citizenshipStatus, and isApplicantApplying did not seem to be implemented yet so I moved them down in the PDF